chain/jsonrpc/src/lib.rs

```diff
@@ -1038,6 +1039,7 @@ pub fn start_http(
     view_client_addr: Addr<ViewClientActor>,
 ) {
     let RpcConfig { addr, cors_allowed_origins, polling_config, limits_config } = config;
+    info!("Starting http server at {}", addr);
```

Comment:
Do we actually need this? cc @frol

Reply:
I moved this out to https://github.com/pulls